test/syncenginetestutils.h

```diff
@@ -14,13 +14,17 @@
 
 #include <QDir>
 #include <QNetworkReply>
+#include <QMap>
 #include <QtTest>
 
 static const QUrl sRootUrl("owncloud://somehost/owncloud/remote.php/webdav/");
 
 static QString generateEtag() {
     return QString::number(QDateTime::currentDateTime().toMSecsSinceEpoch(), 16);
 }
+QByteArray generateFileId() {
```

Comment:
Might need to be static if this header ever gets built more than once per linked program (even though something else will probably fail).